reactive/kotlinx-coroutines-reactive/src/ContextInjector.kt

```diff
@@ -0,0 +1,14 @@
+package kotlinx.coroutines.reactive
+
+import kotlinx.coroutines.InternalCoroutinesApi
+import org.reactivestreams.Publisher
+import kotlin.coroutines.CoroutineContext
+
+/** @suppress */
+@InternalCoroutinesApi
+public interface ContextInjector {
```

Comment:
Could you please make it internal and suppress it in the reactor module?
This API looks dangerous enough not to expose it

Reply:
I'd rather lean into the other direction -- try to stop using supporessions as it teaches the bad precedent and just as easy to suppress from the user code.